pkg/image/definition.go

```diff
@@ -136,9 +136,9 @@ type Network struct {
 }
 
 type Node struct {
-	Hostname string `yaml:"hostname"`
-	Type     string `yaml:"type"`
-	First    bool   `yaml:"firstNode"`
+	Hostname    string `yaml:"hostname"`
+	Type        string `yaml:"type"`
+	Initialiser bool   `yaml:"initialiser"`
```

Comment:
It's not worth blocking the PR for this, but we should discuss the spelling of "initializer" in the image definition (I'm actually being serious this time, not just giving you crap). I think it's potentially confusing that `embeddedArtifactRegistry` uses the Americanized spelling while this uses whatever you call the other spelling version (I know, I know, your answer is going to be "correct"). In the code I don't care if we mix and match, but for a public API I think it looks bad if it's not consistent. We can talk about this in the definition review next week.